conf/i18n/constants.js

```diff
@@ -1,5 +1,110 @@
+exports.TRANSLATION_FLAGGER_REGEX = /TranslationFlagger.flag\((\s)*\{[^;]+?\}(\s)*\)/g;
+
 exports.DEV_LOCALE = 'en';
 
-exports.BUILD_LOCALES = ['en', 'es', 'fr', 'de', 'it', 'ja'];
+exports.DEFAULT_LOCALES = ['en'];
```

Comment:
What's the difference here between `DEFAULT_LOCALES` and `DEV_LOCALE`?

Reply:
`DEV_LOCALE` is the locale used with `npm run dev`. `DEFAULT_LOCALES` are the locales used with `npm run build`. I left `DEFAULT_LOCALES` as an array to allow us to easily create builds for multiple languages. I renamed it from `BUILD_LOCALES` in an attempt to not cause confusion with `ALL_LOCALES` which can also be built. Do you think I should change it back? Or maybe consolidate it with `DEV_LOCALE` ?

Reply:
I think it's confusing to have both `DEFAULT_LOCALES` and `DEV_LOCALE`. Given the two new commands you added (`build locales` and `build languages`), it seems as if `build` will only ever run for a single language. If we eventually want `build` to work across multiple languages/locales, we can revisit. For now, I'd suggest consolidating if possible.

Reply:
I'll consolidate into `DEFAULT_LOCALES`. For the dev builds we just take the first locale from that array.